docs/articles/documentation/using-testcafe/common-concepts/browser-support.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: docs
+title: Browser Support
+permalink: /documentation/using-testcafe/common-concepts/browser-support.html
+---
+# Browser Support
+
+TestCafe allows you to test against different web browsers.
+The most popular browsers are [supported directly](#directly-supported-browsers),
+while others require a [browser provider plugin](#browser-provider-plugins).
+
+The topic contains the following sections.
+
+* [Directly Supported Browsers](#directly-supported-browsers)
+* [Browser Provider Plugins](#browser-provider-plugins)
+    * [Published Browser Provider Plugins](#published-browser-provider-plugins)
+* [Selecting Browsers for Testing](#selecting-browsers-for-testing)
+
+## Directly Supported Browsers
+
+TestCafe supports testing against the following browsers out-of-the-box.
+
+Browser           |Version | Short Browser Name
+----------------- | ------ | -------------------
+Chromium          | 22+    | `chromium`
+Google Chrome     | 22+    | `chrome`
+Internet Explorer | 9+     | `ie`
+Microsoft Edge    |        | `edge`
+Mozilla Firefox   | 17+    | `firefox`
+Opera             | 11+    | `opera`
+Safari            | 6+     | `safari`
+
+## Browser Provider Plugins
+
+If you need to use a browser that is not [supported directly](#directly-supported-browsers), create a [browser provider plugin](../../extending-testcafe/custom-browser-provider-plugin/).
+
+### Published Browser Provider Plugins
+
+After a browser provider plugin is created, it can be published to npm. The provider package name consists of two parts - the `testcafe-browser-provider-` prefix and the name of a provider itself; for example, `testcafe-browser-provider-saucelabs`.
```

Comment:
> After a browser provider plugin is created, it can be published to npm.

This sentence should be worded in a *little bit* different way. Now it looks a little bit alien here. As if it's taken from an instruction, while this paragraph is not an instruction.